app/models/levelling_up_premium_payments/dqt_record.rb

```diff
@@ -13,28 +11,334 @@ class DqtRecord
       to: :record
     )
 
-    def initialize(record, claim)
-      @claim = claim
+    ELIGIBLE_ITT_CODES = [
+      "101038",
```

Comment:
You might want to check with Fisal but there's a possibility of a regular expression which *might* cover exactly this set of codes. For example, it *might* be that *any* code which begins "F1" is eligible. There definitely is a pattern because I believe that's how Fisal created this set in the first place but he might find it hard to describe to you.

Reply:
He didn't mention this when I spoke to him, I'll check.

Reply:
Confirmed by Fisal that the codes have to be harcoded.